apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -1,17 +1,65 @@
 import test from "../../fixtures/common";
```

Comment:
general question - why we every time start send flow from account page and never from Send button in the menu?